scripts/trainer.py

```diff
@@ -59,7 +59,8 @@ def go(self, args):
 
         model_file_name = f"{model_name}model"
```

Comment:
Since it is not a file anymore, we may need to rename it (even in other places):
```suggestion
        model_dir = f"{model_name}model"
```